hilti/toolchain/include/ast/type.api

```diff
@@ -2,6 +2,9 @@
 
 /** Interface for HILTI types. */
 class Type(hilti::trait::isType) : hilti::trait::isNode {
+    /** Returns the number of bytes the type consumes if known. */
```

Comment:
Wondering what the right phrasing for the description here is. We are in HILTI, so "consume" doesn't match. Maybe: "number of bytes of the type's binary serialization if that's statically determinable." (but also see below on whether we want to even keep this).

Reply:
This change is gone now I dropped adding `size()` to `Type`.